osmosis-1/osmosis-1.assetlist.json

```diff
@@ -12926,11 +12928,7 @@
       ],
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/baddog.png"
-      },
-      "keywords": [
-        "osmosis-info",
-        "osmosis-price:ibc/B9E0A1A524E98BB407D3CED8720EFEFD186002F90C1B1B7964811DD0CCC12228:1356"
```

Comment:
you changed $baddog CA and now isn't showing up on front end.
correct CA is ibc/2FFE07C4B4EFC0DDA099A16C6AF3C9CCA653CC56077E87217A585D48794B0BC7

Reply:
Or maybe that is ibc channel for 1356. either way its not appearing on frontend because of this auto mod merge. Please get back to me.